apps/root/src/frame/polling-handlers/index.tsx

```diff
@@ -10,7 +10,7 @@ const PollingHandlers = () => (
     <TransactionUpdater />
     <BalancesUpdater />
     <NetworkUpdater />
-    <EarnPositionsUpdater />
+    {(process.env.EARN_ENABLED === 'true' && <EarnPositionsUpdater />) ?? null}
```

Comment:
I would replace `??` with another operator, the first expression will never be null/undefined